### ISSUE TITLE
Modify Readme.md

### ISSUE THREAD

**Issue by DeepNinja07x** [DeepNinja07x/Python_Scripts#3]
After successfully creating a PR add your name along with your GitHub handle and profile picture in the __Contributors Wall__ section. Follow the layout while adding your details.
Click on [README.md](https://github.com/DeepNinja07x/Python_Scripts/blob/master/README.md) to directly go to the file.

**Comment by utsavgadhiya** [comment]
Hi, @DeepNinja07x , Can you assign me this one ?

**Comment by DeepNinja07x** [comment]
Yeah, sure work on this one @utsavgadhiya

**Comment by sarthak1905** [comment]
@DeepNinja07x  Hey! My PR was successfully merged and closed. Could I be assigned this to add my name to the contributors wall?

Thanks and Regards
Sarthak Saxena

**Comment by DeepNinja07x** [comment]
Yeah sure, I even mentioned you to do this in your previous PR. @sarthak1905
